frontend/src/components/common/SortTablePopover.tsx

```diff
@@ -0,0 +1,134 @@
+import React from "react";
```

Comment:
delete this file pls - it was removed in this pr https://github.com/uwblueprint/jump-math/pull/116